lib/kitchen/directions/bake_chapter_glossary.rb

```diff
@@ -5,35 +5,40 @@ module Directions
     # Bake directons for eoc glossary
     #
     module BakeChapterGlossary
-      def self.v1(chapter:, metadata_source:, append_to: nil)
-        metadata_elements = metadata_source.children_to_keep.copy
+      def self.v1(chapter:, metadata_source:, append_to: nil, uuid_prefix: nil)
+        V1.new.bake(
+          chapter: chapter,
+          metadata_source: metadata_source,
+          append_to: append_to,
+          uuid_prefix: uuid_prefix
+        )
+      end
 
-        definitions = chapter.glossaries.search('dl').cut
-        definitions.sort_by! do |definition|
-          [definition.first('dt').text.downcase, definition.first('dd').text.downcase]
-        end
+      class V1
+        renderable
+        def bake(chapter:, metadata_source:, append_to:, uuid_prefix:)
+          @metadata = metadata_source.children_to_keep.copy
+          @klass = 'glossary'
+          @title = I18n.t(:eoc_key_terms_title)
+          @uuid_prefix = uuid_prefix
 
-        chapter.glossaries.trash
-
-        return if definitions.none?
-
-        append_to_element = append_to || chapter
-        title_tag = append_to ? 'h3' : 'h2'
-
-        append_to_element.append(child:
-          <<~HTML
-            <div class="os-eoc os-glossary-container" data-type="composite-page" data-uuid-key="glossary">
-              <#{title_tag} data-type="document-title">
-                <span class="os-text">#{I18n.t(:eoc_key_terms_title)}</span>
-              </#{title_tag}>
-              <div data-type="metadata" style="display: none;">
-                <h1 data-type="document-title" itemprop="name">#{I18n.t(:eoc_key_terms_title)}</h1>
-                #{metadata_elements.paste}
-              </div>
-              #{definitions.paste}
-            </div>
-          HTML
-        )
+          definitions = chapter.glossaries.search('dl').cut
+          definitions.sort_by! do |definition|
+            [definition.first('dt').text.downcase, definition.first('dd').text.downcase]
+          end
+
+          chapter.glossaries.trash
+
+          return if definitions.none?
```

Comment:
I'd move this right after line 25.  If there are no definitions, we don't need to sort them.  Also move `chapter.glossaries.trash` above setting `definitions` b/c it isn't related to these lines of code, seems out of place wandering here in the midst of them.